src/main/groovy/nebula/plugin/publishing/ivy/IvyResolvedDependenciesPlugin.groovy

```diff
@@ -59,7 +59,11 @@ class IvyResolvedDependenciesPlugin implements Plugin<Project> {
                                 if (!resolved) {
                                     return  // continue loop if a dependency is not found in dependencyMap
                                 }
-                                dep.@rev = resolved?.selected?.moduleVersion?.version
```

Comment:
@rspieldenner any idea what why all the `?.`ing was required? If we reach here, they should always be non-null, no?